src/actions.js

```diff
@@ -0,0 +1,112 @@
+import { MAX_PAGE } from "./constants.js";
+import { fetchPressData, fetchNewsData, fetchHotTopicData } from "./utils.js";
+
+export function useRenderOptions(type, is_sub) {
+    if (type === "list") {
```

Comment:
list나 grid가 자주 사용되는것 같아서 이런식으로 관리하면 좋을것 같아요!

```suggestion
const mode = {
  grid:'grid',
  list:'list'
}
```